_data/projects.yaml

```diff
@@ -83,3 +83,18 @@
   url: https://www.shopsys-framework.com/
   when: Fall 2018
   done: false
+  
+- name: PHP-PM
+  url: https://github.com/php-pm/php-pm/pull/335
```

Comment:
PHP 7.0 only. 

Also what happens if this PR won't be merged?

Only accepted PRs